dedupe/record_linkage_testing/compose.yml

```diff
@@ -0,0 +1,64 @@
+version: "3.8"
```

Comment:
nit: the new version of docker Compose does not require `version` to be included in the compose file anymore.

Reply:
Correct, it's just informational.  My thought was that it provides some context as to what version of the spec we're writing against at the time, but I can remove if you think that might cause confusion.

https://github.com/compose-spec/compose-spec/blob/master/04-version-and-name.md#version-top-level-element

Reply:
Nope, I don't think it'll cause confusion. Any comment that I prefix with `nit` is just my own preference and can be discarded as you wish.